README.md

```diff
@@ -30,3 +30,7 @@ Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Con
 ## Contact
 
 You can reach the admins, maintainers, and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or interact with the community. Join Us!
+
+## License
+
+Open Source Programs project is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](https://github.com/anitab-org/open-source-programs-backend/blob/develop/LICENSE) file.
```

Comment:
```suggestion
Open Source Programs project is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](LICENSE) file.
```
It was actually find because the relative link helps making this link accessible in multiple forks. So by having "license" only, Github will render the correct license when clicking that link.
For example, in my fork it would redirect to https://github.com/isabelcosta/open-source-programs-backend/blob/develop/LICENSE instead of https://github.com/anitab-org/open-source-programs-backend/blob/develop/LICENSE :)
So using relative paths will allow correct render of the link, regardless of branch, and if the repo is a fork cc @Amulya-coder @ayan-biswas0412 
Can you please change it back?

Reply:
Yes! I will do that